sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryAsyncClient.java

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.containers.containerregistry;
+
+import com.azure.containers.containerregistry.implementation.ContainerRegistriesImpl;
+import com.azure.containers.containerregistry.models.AcrErrorsException;
+import com.azure.containers.containerregistry.models.DeleteRepositoryResult;
+import com.azure.containers.containerregistry.models.ListRepositoriesOptions;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.rest.PagedFlux;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.Context;
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Mono;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.azure.core.util.FluxUtil.monoError;
+import static com.azure.core.util.FluxUtil.withContext;
+
+/**
+ * Initializes a new instance of the asynchronous ContainerRegistry type.
+ */
```

Comment:
This seems to be a Javadoc copied from the builder.

Reply:
This actually got added as part of the codegen when I tried generating public clients, which I later realized is not the standard practice and we tend to hand write our public clients. So I need to update pretty much all our javadocs :).

I will be sending a separate PR for those.

---
In reply to: [602629563](https://github.com/Azure/azure-sdk-for-java/pull/20139#discussion_r602629563) [](ancestors = 602629563)